clusterloader2/pkg/measurement/common/metrics_for_e2e.go

```diff
@@ -38,23 +38,106 @@ var interestingApiServerMetricsLabels = []string{
 }
 
 var interestingControllerManagerMetricsLabels = []string{
```

Comment:
Thanks Sonya!

Looking at your PR it got me thinking whether we need this "interesting" metric logic.
The cons of it are that we need to keep this list up2date which has non-zero maintanance costs (e.g. someone adds new metric or metric gets replaced by other and we need to update it here).
And when it comes to pros  the only one I see is less storage used for a test run results but these savings are in KB (maybe small MB) and comparing to GBs of logs a single run can generate it doesn't really matter.

Given that, could you remove this "interesting...MetricsLables and simply make this code dump all metrics? Let's start with KubeControllerManager, we'll see how it goes and if I haven't missed anything we can later change it for other components.

Thanks!

Reply:
I agree with you and will submit new update soon.